samples/net/sockets/big_http_download/prj.conf

```diff
@@ -27,9 +27,6 @@ CONFIG_NET_CONFIG_NEED_IPV4=y
 CONFIG_NET_CONFIG_MY_IPV4_ADDR="192.0.2.1"
 CONFIG_NET_CONFIG_PEER_IPV4_ADDR="192.0.2.2"
 CONFIG_NET_CONFIG_MY_IPV4_GW="192.0.2.2"
-# DHCP configuration. Until DHCP address is assigned,
-# static configuration above is used instead.
```

Comment:
This comment effectively sets the requirement for networking stack configuration, and this sample effectively verifies this requirement. And the current state that we have a regression in the behavior. So, please don't remove this comment, but instead investigate what/when broke it. (I can help with that too, as it was in my backlog for some time.)